app/build.gradle

```diff
@@ -69,6 +70,10 @@ dependencies {
     implementation "com.squareup.retrofit2:converter-moshi:2.4.0"
     implementation "com.github.skydoves:landscapist-glide:1.4.5"
     implementation "io.coil-kt:coil-compose:1.3.2" // 1.3.2
+    implementation 'com.google.code.gson:gson:2.8.7'
+    implementation  "androidx.datastore:datastore-core:1.0.0"
+    implementation  "com.google.protobuf:protobuf-javalite:3.18.0"
+    implementation("androidx.datastore:datastore-preferences:1.0.0")
```

Comment:
I love how there's 3 different coding conventions across all 4 lines. 😆 
Line 73: Single Quotes
Line 74-75: Double Quotes, extra space before
Line 76: Parenthesis, double quotes, no space.

Can we change these to:
```
implementation "com.google.code.gson:gson:2.8.7"
implementation "androidx.datastore:datastore-core:1.0.0"
implementation "com.google.protobuf:protobuf-javalite:3.18.0"
implementation "androidx.datastore:datastore-preferences:1.0.0"
```